DOCUMENTATION.md

```diff
@@ -482,10 +502,81 @@ However, we believe that they are fairer and well-supported by research in machi
+
+Submitters may submit algorithms marked as *baselines*. These might include existing algorithms with different search spaces or learning rate schedules. These baseline algorithms are not eligible for winning the competition or prize money but they are also not required to be "substantially different" from other submissions by the same submitters.
+
+#### Can my submission be structured using multiple files?
+
+Yes, your submission can be structured using multiple files.
+
+#### Can I install custom dependencies?
+
+You may use custom dependencies as long as they do not conflict with any of the pinned packages in `algorithmic-efficiency/setup.cfg`.
+To include your custom dependencies in your submission, please include them in a requirements.txt file. Please refer to the [Software dependencies](#software-dependencies) section of our rules.
+
+#### How can I know if my code can be run on benchmarking hardware?
+
+The benchmarking hardware specifications are documented in the [Benchmarking Hardware Section](#benchmarking-hardware). We recommend monitoring your submission's memory usage so that it does not exceed the available memory
+on the benchmarking hardware. We also recommend to do a dry run using a cloud instance.
+
+#### Are we allowed to use our own hardware to self-report the results?
+
+You only have to use the benchmarking hardware for runs that are directly involved in the scoring procedure. This includes all runs for the self-tuning ruleset, but only the runs of the best hyperparameter configuration in each study for the external tuning ruleset. For example, you could use your own (different) hardware to tune your submission and identify the best hyperparameter configuration (in each study) and then only run this configuration (i.e. 5 runs, one for each study) on the benchmarking hardware.
+
+#### What can I do if running the benchmark is too expensive for me?
+
+Submitters unable to self-fund scoring costs can instead self-report only on the [qualification set of workloads](/COMPETITION_RULES.md#qualification-set) that excludes some of the most expensive workloads. Based on this performance on the qualification set, the working group will provide - as funding allows - compute to evaluate and score the most promising submissions. Additionally, we encourage researchers to reach out to the [working group](mailto:algorithms@mlcommons.org) to find potential collaborators with the resources to run larger, more comprehensive experiments for both developing and scoring submissions.
+
+#### Can I submit existing (i.e. published) training algorithms as submissions?
+
+Yes you may, as long as it isn't an exact copy of an existing submission.
+For example, you may submit the Adam optimizer with your hyperparameters or hyperparameter search spaces, as this constitues a different training algorithm.
```

Comment:
@georgedahl Can you double check the answer to this question? I think its phrasing could be improved.

Reply:
How about:
"Yes you may, as long as it isn't an exact copy of an existing submission.
For example, you may submit the Adam optimizer with your particularly effective hyperparameter search space and hyperparameter configuration, as different choices for hyperparameter values and/or search spaces constitute different training algorithms and are potential sources of innovation." 

If you want, we could also add a bit more text along the following lines (feel free to not add it, but if you do fill in the link to the other question):

"That said, while submitting Adam with some novel heuristic to set various hyperparameters, some especially effective hyperparameter search space, or your single best hyperparameter configuration is fine, avoid making *multiple* submissions that only differ by their hyperparameter configuration without a convincing justification they are substantially different (see "Can I submit multiple times to the benchmark competition?", above)."